packages/core/addon/models/bard-request-v2/fragments/base.ts

```diff
@@ -85,4 +86,17 @@ export default class Base<T extends ColumnType> extends Fragment.extend(Validati
     const { parameters, columnMetadata } = this;
     return this.naviFormatter.formatColumnName(columnMetadata, parameters, null);
   }
+
+  @computed('columnMetadata')
+  get filterBuilderDisplayName() {
```

Comment:
Since this model is a general purpose object, I think this logic belongs closer to the filter component.